cognite/client/data_classes/aggregations.py

```diff
@@ -213,10 +213,6 @@ class AggregationFilter(ABC):
     def dump(self) -> dict[str, Any]:
         return {self._filter_name: self._filter_body()}
 
-    @classmethod
-    def load(cls, filter_: dict[str, Any]) -> AggregationFilter:
```

Comment:
Why removing this?

Reply:
It is not implemented in any subclasses (and thus invalid for subclasses using `@final`).

```
cognite/client/data_classes/aggregations.py:217: error: Missing return
statement  [empty-body]
        def load(cls, filter_: dict[str, Any]) -> AggregationFilter:
        ^
cognite/client/data_classes/aggregations.py:217: note: If the method is meant to be abstract, use @abc.abstractmethod
```
****
#### Also:

<img width="866" alt="image" src="https://github.com/cognitedata/cognite-sdk-python/assets/8521241/08c5fb1d-95e6-46b9-9dfd-f8ca77b1b285">

Reply:
Makes sense